src/__tests__/cryptoKey.test.ts

```diff
@@ -35,6 +35,26 @@ describe("Generating key pair", () => {
     });
 });
 
+describe("Generating key pair from seperate keys", () => {
+    it("Should succeed with valid public and pirvate key strings", () => {
+        const keyPair = cryptoKeyUtil.generateKeyPair();
+
+        const result = cryptoKeyUtil.verifyKeyPair(keyPair.getPublic(true, "hex") as string, keyPair.getPrivate("hex"));
```

Comment:
Make a const for "hex", so you don't need to change it on 2 places when there is need to change it